src/EPR.Calculator.Frontend/Program.cs

```diff
@@ -22,6 +22,7 @@
 if (!app.Environment.IsDevelopment())
 {
     app.UseExceptionHandler("/Dashboard/Error");
+
```

Comment:
Can you remove this format line to see a change on this file?

Reply:
Hi @udaydefra, the StyleCop is not allowing for that blank line to be removed. The line after this is a `// comment`, and StyleCop says that there should be a blank line before `// comment`.